stages/02_process_embl.py

```diff
@@ -0,0 +1,11 @@
+from Bio import SeqIO
+import pandas as pd
+
+new_embl_records = SeqIO.parse("./download/miRNA.dat", "embl")
+# prev_embl_records = SeqIO.parse("./download/miRNA.dead", "embl")
+records = [{'name': record.name, 'description': record.description} for record in new_embl_records]
```

Comment:
These are good for now. I think adding the sequence would also be helpful.

There is more that needs to be done with respect to getting the miRNA features that make up the stem-loops. This is why having the original files under `download` is still useful as the specialised parsers can get this information more easily.